assets/models/system/bert-large-uncased/spec.yaml

```diff
@@ -24,28 +24,6 @@ properties:
   languages: en
 tags:
   Preview: ''
-  computes_allow_list:
```

Comment:
Same here

Reply:
We have ` finetune-min-sku-spec: 4|1|28|176` in properties. So we don't need to maintain he allow list.